packages/expo-updates/ios/EXUpdates/ReactDelegateHandler/ExpoUpdatesAppDelegateSubscriber.swift

```diff
@@ -15,4 +15,9 @@ public final class ExpoUpdatesAppDelegateSubscriber: ExpoAppDelegateSubscriber {
     }
     return true
   }
+
+  public func applicationWillEnterForeground(_ application: UIApplication) {
```

Comment:
this should be in a separate PR I think?

Reply:
Agreed, this was mainly to test if `checkForUpdate()` can be called from outside JS in response to a notification.